Frontend/ui-common/src/Components/ChooseVehicle/View.purs

```diff
@@ -77,13 +77,14 @@ vehicleDetailsView push config =
         , cornerRadius 2.5
         , background Color.black600
         , margin (Margin 5 12 0 0)
+        , visibility if config.isBookingOption then GONE else VISIBLE
         ]
         []
     , linearLayout
         [ width WRAP_CONTENT
         , height WRAP_CONTENT
         , orientation HORIZONTAL
-        , margin (Margin 5 5 0 0)
+        , margin $ if config.isBookingOption then (Margin 0 0 0 0) else (Margin 5 5 0 0)
```

Comment:
instead of using a variable for this we can make it configurable. The margin itself can be made configurable.